packages/components-native/src/AtlantisContext/AtlantisContext.tsx

```diff
@@ -2,6 +2,8 @@
 import { createContext, useContext } from "react";
 import RNLocalize from "react-native-localize";
 
+export const DEFAULT_CURRENCY_SYMBOL = "$";
```

Comment:
I wonder if this constant should sit somewhere more relevant... maybe in InputCurrency (the component or a constants file in that folder)?